voss-25: update styles to match oss template

amymanion opened the pull request:
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**
- Fixes #

**Special notes for your reviewer**:

**Release note**:
```
release-note
```

amymanion committed:
voss-25: update styles to match oss template

ccbayer commented on site/_layouts/docs.html:
```diff
@@ -19,7 +19,7 @@
         <div class="section">
             <div class="section-content">
-              <h1>Documentation</h1>
+              <h2>Documentation</h2>
```
@amymanion this should be an h1, as it is the title of the page. if it needs to look different use CSS

ccbayer commented on site/_layouts/page.html:
```diff
@@ -9,7 +9,7 @@
         <div class="section">
             <div class="section-content">
-            <h1>{{ page.title }}</h1>
+            <h2>{{ page.title }}</h2>
```
@amymanion this should be an h1, as it is the title of the page. if it needs to look different use CSS

ccbayer commented on site/_layouts/posts.html:
```diff
@@ -9,7 +9,7 @@
         <div class="section">
           <div class="section-content">
-            <h1>{{ page.title }}</h1>
+            <h2>{{ page.title }}</h2>
```
@amymanion this should be an h1, as it is the title of the page. if it needs to look different use CSS

ccbayer commented on site/_scss/site/objects/_footer.scss:
```diff
@@ -25,9 +26,10 @@
-    color: $footer-copyright;
+    font-size: 0.75rem;
+    display: inline;
+    //font-weight: $font-weight-light;
```
@amymanion please remove commented line

ccbayer commented on site/_scss/site/objects/_footer.scss:
```diff
@@ -1,3 +1,4 @@
+@import 'mixins';
```
@amymanion don't import mixins to another partial; instead import it into `site/_scss/_styles.scss`

ccbayer commented on site/_scss/site/objects/_components.scss:
```diff
@@ -0,0 +1,136 @@
+@import 'mixins';
```
@amymanion don't import mixins to another partial; instead import it into site/_scss/_styles.scss

ccbayer commented on site/_scss/site/objects/_mixins.scss:
```diff
@@ -0,0 +1,14 @@
+@mixin breakpoint($point) {
```
@amymanion do you need this? because there is a built in bootstrap mixin you could use instead:

```
@include media-breakpoint-up($size) { ... }
```

ccbayer commented on site/blog.html:
```diff
@@ -7,11 +7,11 @@
   <div class="section">
     <div class="section-content">
-      <h1>{{ page.title }}</h1>
+      <h2>{{ page.title }}</h2>
```
@amymanion this should be h1

ccbayer commented on site/resources.html:
```diff
@@ -0,0 +1,48 @@
+---
+
+<div class="sub-page">
+    <h3>Here you will find external resources about Sonobuoy, such as videos, podcasts, and community articles.</h3>
```
@amymanion should this be h2?

ccbayer commented on site/resources.html:
```diff
@@ -0,0 +1,48 @@
+    <h3>Here you will find external resources about Sonobuoy, such as videos, podcasts, and community articles.</h3>
+    <div class="grid three">
+        <div class="col">
+            <p><iframe width="100%" height="auto" src="https://www.youtube.com/embed/1e6SAZfkqUk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
```
@amymanion use bootstrap responsive embed instead:

```
<div class="embed-responsive embed-responsive-16by9">
  <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/1e6SAZfkqUk"></iframe>
</div>
```

ccbayer reviewed (changes requested):
@amymanion see comments

amymanion committed:
Code updates
-fix headings
-fix footer
-use bootstrap breakpoints
-fix iframe to use bootstrap embed

ccbayer commented on site/README-JEKYLL.md:
```diff
@@ -22,8 +22,8 @@ This mirrors the plug-ins used by GitHub Pages on your local machine including J
 4. `rbenv local 2.6.3`
-5. `bundle install`
-6. Serve the site and watch for markup/sass changes `jekyll serve --livereload`. You may need to run `bundle exec jekyll serve --livereload`.
+5. `rbenv exec bundle install`
```
@amymanion i update this note on all three projects to read,

```
4. If using `rbenv`, set to `rbenv local 2.6.3`
5. `bundle install` - if using `rbenv`, you will need to run `rbenv exec` before doing any Ruby commands! e.g, `rbenv exec bundle install` or `rbenv exec jekyll serve --livereload`
6. Serve the site and watch for markup/sass changes ` jekyll serve --livereload`. 
```

ccbayer commented on site/_layouts/tag-index.html:
```diff
@@ -7,7 +7,7 @@
     <div class="section">
         <div class="section-content">
-        <h1>{{ page.title }}</h1>
+        <h2>{{ page.title }}</h2>
```
@amymanion indent

ccbayer commented on site/_scss/site/layouts/_container.scss:
```diff
@@ -1,5 +1,5 @@
 .site-outer-container {
-  max-width: $container-max-width;
+  //max-width: $container-max-width;
```
@amymanion remove commented code

ccbayer commented on site/_scss/site/objects/_components.scss:
```diff
@@ -0,0 +1,142 @@
+.grid.three {
```
@amymanion as with others, break this into separate partials for each "component" and define variables for hex values used more than once

ccbayer commented on site/community.html:
```diff
@@ -0,0 +1,38 @@
+---
+
+<div class="sub-page">
+    <h2>Do you want to help build Sonobuoy?</h2>
```
@amymanion shouldn't this be h1?

amymanion replied:
@ccbayer No, there is a "Community" header at the top of the page which is the h1.

ccbayer commented on site/community.html:
```diff
@@ -0,0 +1,38 @@
+            </div>
+            <div class="content">
+                <p>If you’re a newcomer, check out the "<a href="https://github.com/vmware-tanzu/sonobuoy/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+first+issue%22">Good first issue</a>" and "<a href="https://github.com/vmware-tanzu/sonobuoy/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3A%22Help+wanted%22+">Help wanted</a>" labels in the Sonobuoy repository.</p>
+                <p>Join the <a href="https://github.com/projectcontour/community/wiki/Office-Hours">Contour Office Hours</a>, a place where you can ask questions like “how do I do this in Contour” and “why does Contour do this thing this way?”</p>
```
@amymanion double check this link, it looks like it goes to contour

ccbayer commented on site/plugins.html:
```diff
@@ -0,0 +1,27 @@
+  </div>
+</div>
+
+<!-- Sonobuoy was always designed to facilitate third-party plugins in order to accommodate custom testing requirements, and recently, the work on Sonobuoy made some advanced plugins possible to create.
```
@amymanion remove commented code

ccbayer commented on site/resources.html:
```diff
@@ -0,0 +1,54 @@
+---
+
+<div class="sub-page">
+    <h2>Here you will find external resources about Sonobuoy, such as videos, podcasts, and community articles.</h2>
```
@amymanion shouldn't this be h1?

amymanion replied:
@ccbayer No, there is a "Resources" header at the top of the page which is the h1.

ccbayer commented on site/resources.html:
```diff
@@ -0,0 +1,54 @@
+                <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/Aqp6Rk1J5Jw"></iframe>
+            </div>
+            <div class="content">
+                <p><a href="https://www.youtube.com/embed/Aqp6Rk1J5Jw">Sonobuoy in 10 minutes, presented at Kubernetes Community Meeting</a></p>
```
@amymanion this shouldn't link to the embed url but should link to the video link on youtube?

amymanion replied:
Fixed this.

ccbayer commented on site/resources.html:
```diff
@@ -0,0 +1,54 @@
+                <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/k-P4hXdruRs"></iframe>
+            </div>
+            <div class="content">
+                <p><a href="https://www.youtube.com/embed/k-P4hXdruRs">Overview of Sonobuoy</a></p>
```
@amymanion this shouldn't link to the embed url but should link to the video link on youtube?

amymanion replied:
Fixed this.

ccbayer commented on site/resources.html:
```diff
@@ -0,0 +1,54 @@
+                <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/1e6SAZfkqUk"></iframe>
+            </div>
+            <div class="content">
+                <p><a href="https://www.youtube.com/embed/1e6SAZfkqUk">A first look at Sonobuoy</a></p>
```
@amymanion this shouldn't link to the embed url but should link to the video link on youtube?

amymanion replied:
Fixed this.

ccbayer reviewed (changes requested):
@amymanion see comments

amymanion committed:
Fix youtube links & add color variables

A-P committed:
Initial Hugo setup and docs port

A-P committed:
Fix broken links in docs markdown

A-P committed:
Add external links to docs

amymanion committed:
Voss-25: Move content from Jekyll to Hugo

amymanion committed:
Voss-25: mobile header update and blog tags link

amymanion committed:
Voss-25: Update config & tag

amymanion committed:
Voss-25: Update author tags, getting started link

A-P committed:
updates to blog feed

A-P committed:
Merge branch 'voss-25-oss-updates' of https://github.com/smalltalk-ai/sonobuoy into voss-25-oss-updates

A-P committed:
update feed URL

A-P committed:
Fix hero docs link

amymanion committed:
voss-25: update h2 color

A-P committed:
Merge branch 'voss-25-oss-updates' of https://github.com/smalltalk-ai/sonobuoy into voss-25-oss-updates

A-P committed:
Move hugo site to main /site folder

A-P committed:
Rename logo and update readme

A-P committed:
Fix content on communities page

A-P committed:
Update docs styling

A-P committed:
Change the homepage hero image style

A-P committed:
Update homepage hero image styling

Commit:
updates based on https://github.com/vmware-tanzu/pinniped/pull/426

michow committed:
Merge pull request #2 from smalltalk-ai/VOSS-40-accessibility-updates

updates based on https://github.com/vmware-tanzu/pinniped/pull/426

Commit:
updates based on feedback

Commit:
change code color

michow committed:
Merge pull request #3 from smalltalk-ai/VOSS-41-final-tweaks

updates based on feedback

Commit:
Blog slug permalinks

Commit:
Move imagery to static

lspringer committed:
Merge pull request #4 from smalltalk-ai/PATHFIX

Pathfix
